connector/clients/kutis_client.py

```
@@ -0,0 +1,96 @@
+    KUTIS_LOGIN_URL,
+    KUTIS_MAIN_PAGE_URL,
+)
+from connector.exception import ClientException, ErrorType
+
+
+class KutisClient:
+    def __init__(self, studentId: str, password: str):
+        self.studentId, self.password = studentId, password
+        self._session: aiohttp.ClientSession | None = None
+
+    async def login(self) -> "KutisClient.session":
```

Ruff (0.11.9)

`login` is too complex (12 > 10)

(C901)